gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/ComposerTest.java

```diff
@@ -53,8 +56,13 @@ class ComposerTest {
           .build();
   private List<Sample> ListofSamples = Arrays.asList(new Sample[] {sample});
 
+  @Before
```

Comment:
Could you change this tag to `@BeforeEach`?

Reply:
Done